gluon/custom_import.py

```diff
@@ -67,6 +67,13 @@ def custom_importer(name, globals=None, locals=None, fromlist=None, level=-1):
         import_tb = None
         try:
             try:
+                if ('gluon' not in name and not name.startswith('.') and
```

Comment:
You can save one condition: if name.split('.')[0] == 'gluon' ... Something
like this...
Em 23/01/2015 18:59, "mdipierro" <notifications@github.com> escreveu:

> In gluon/custom_import.py
> <https://github.com/web2py/web2py/pull/587#discussion_r23479800>:
>
> > @@ -67,6 +67,13 @@ def custom_importer(name, globals=None, locals=None, fromlist=None, level=-1):
> >          import_tb = None
> >          try:
> >              try:
> > +                if ('gluon' not in name and not name.startswith('.') and
>
> I this if name does not start with 'gluon.' and does not start with '.'
> should try name = 'gluon.'+name but we should not check for "gluon" in name.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/web2py/web2py/pull/587/files#r23479800>.
>